### DISCUSSION TITLE
16還18周

### DISCUSSION THREAD

**Discussion by Cerosop** [1101-GS4719-C/Python-Community-GS4719-C#1812] [General]
從2022開始，以台大為首很多學校都開始了只有16周的學期，只剩下很多16周的學生說時間不夠，十分繁忙，但維持18周的學生又說希望改成16周以增加寒暑假時間，甚至還有更奇怪的16+2周跑出來，導致學生要背負部分16周的繁忙又沒辦法提早規劃想做的事。大家覺得哪個比較好呢，然後為甚麼。

**Comment by Cerosop** [comment]
我自己是16周派的，因為我想要長一點的假期，而且我發覺其實學期初跟學期中的時候很閒，是可以被壓縮的。

**Comment by spermwhale464** [comment]
16周 我想放假

**Comment by uwje** [comment]
我認為不管那個只要統一就好，不要自由心證，導致期末考週原本一週結束變成要三週，也就是16-18都是期末考週，導致我們要有更長時間處在考試的壓力下，其實16 18週各有優缺，也希望中央大學能盡快決定哪個好，也可以讓使用16週 並提供通識期末作業能延長繳交時間來避免卡到考試

**Comment by Patsy17** [comment]
16週～但如果老師教不完後期變得很趕的話，會比較希望18週才不會事情都卡在一起

**Comment by sorn32** [comment]
希望統一16週，分散開來導致一個禮拜只考一科真的很煩(╯‵□′)╯︵┴─┴

**Comment by 12Jesse24** [comment]
支持16周，想趕快解脫考試一堆的期末周。分散開來有的很早考完有的壓在18周可能會導致一個禮拜只需準備一科，就會有那種放假卻還要考試的錯覺，不過確實不管16周、18周都各有優缺。

**Comment by 1chooo** [comment]
支持 16 週，但絕對不要 16 ~ 18 彈性，這樣每個 Deadline 都不一樣，對我而言反而無法調適，發揮彈性真正的意義！

**Reply by chia-kai-chang** [reply]
沒錯，所以我下學期要嚴格執行16+2，而且盡量在第16週完成所有考試、作業、報告等。以前我都有點怕怕，擔心別的老師還在上課，我已經放假了。但是最近發現，其實有很多老師都改成16+2了。我目前雖然也是16+2，但是仍有報告、討論區，壓在第18週。我下學期要嚴格壓在16週。謝謝助教這學期的協助。

**Comment by alvin119** [comment]
還是希望18週。當然也很希望趕快放假，但如果是16週的話就期末就很容易會全部考試擠在一起，
導致某一些科目可能會被放棄，最後讓分數變得不好看。

**Comment by chia-kai-chang** [comment]
我下學期要嚴格執行16+2。盡量讓大家在第16週就完成所有考試，報告。

**Reply by yubo0oo** [reply]
德政

**Comment by B1nn9** [comment]
比較偏好18週，因為這學期修的兩堂必修最後都還有進度沒趕完就直接放掉然後考期末了，雖然有興趣的人還是可以自己額外去花時間補，但整體會有種沒把這門科目修完整的感覺。

**Comment by chuanmin0317** [comment]
雖然現在學校是執行16+2不過從大一上到現在，每次都會有到最後一周的考試，從來沒有提前放假過，反而有一些課為了在時間內結束，搞得我很忙，所以我覺得還是要統一規定不要有彈性比較好

**Comment by chiuwangwang** [comment]
我覺得16+2不錯，但前提是+2裡不能有課程教學，因為我認為+2是給學生與老師交流的彈性，對於課程要有更進一步的了解，我認為+2是非常好的辦法

**Reply by 20050201Shawn** [reply]
我也這麼認為，對理工科系來說其實16周蠻趕的，希望正課盡量16周上完，剩下兩周可以考期末考或是補交作業等，對成績有疑慮也可以跟老師有溝通的彈性。

**Comment by lyykey** [comment]
個人覺得，要統一學校的週數，不要有所謂的彈性2周，因為可能 A B C D E F老師覺得16周課程結束放假，有某一個 G 老師硬要到18周考試，這樣其實讓學校執行所謂的 16+2 意義不大，因為還是要考試，基本上無法停下來休息，或是做自我學習

**Comment by benbebobo149** [comment]
對我來說，我更傾向於16周的學期制度。這樣的制度給予了學生更多的時間去追求自己的興趣和活動，同時也提供了更多的彈性，讓我們能夠更好地平衡學業和個人生活。然而，學校需要確保適當的支援和輔導，以應對學生在課程安排上可能遇到的挑戰。感謝同學分享這個議題，這讓我們能夠思考並討論學期制度的不同影響和利弊。

**Comment by YifenLin** [comment]
我自己的觀點反而跟大家都不同，我更喜歡16~18週，這樣事情才不會擠在一起，準備起來比較有餘裕，也比較不會最後一個禮拜爆肝，可以在準備報告/考試之餘安排運動或休閒等等。

**Comment by SusuTima** [comment]
我喜歡16、17、18週並行，不要強制16週，有些科目本身學習量很大，有些甚至會考到4次段考，如果強迫改成16週的話，會有種快要窒息的感覺。而科目學習量較小的改成16週就比較可以接受，也才不會全部考試堆在一起，一個禮拜一次考好幾科。

**Comment by KH1126** [comment]
我比較喜歡彈性16-18周，因為每個教授可以自己做決定，有些教授會在16周結束，一些在18周結束。通過這樣可以使考試叫分散，而不會全部都擠在一周考試。
因此我比較喜歡彈性16-18周。

**Comment by YANRONG0123** [comment]
我認為有些簡單的科目，16週是沒什麼問題，因為很多資料感覺是為了補充而補充，其實自己回家看也可以，但是蠻多必修課，都是需要完整的18週，才能夠很緊迫的上完，所以我支持16~18週共同實施，才能夠在16週的課程結束後，全心全意拚18週的課程。

**Comment by LinChloe** [comment]
我比較喜歡彈性16-18周，因為每個教授可以自己做決定，有些教授會在16周結束，一些在18周結束。通過這樣可以使考試叫分散，而不會全部都擠在一周考試。
因此我比較喜歡彈性16-18周。

**Comment by yxin214** [comment]
我喜歡16+2週，雖然有些人可能會覺得這會導致各科期末考考試時間不一，但相對的也不會讓每一科期末考都擠在同一週，降低負擔。而且若實施16+2週，其實那17、18週也沒什麼課程，就來考考試而已，基本上跟放假一樣。

**Comment by wanka2030** [comment]
認真覺得當大家都是16週的狀況之下，應該要讓大家都習慣十六週的進度，時間的安排之下可以讓學生有更多的時間可以規劃自己的寒假計畫，根據自己的規劃去執行許多活動，而非單純的限制學生活在刻板的制度之下。

**Comment by Eileen050824** [comment]
雖然16週比較輕鬆(畢竟早放假誰不開心๑> ₃ <)，但坦白講我認為18週還是最適合的！就像108課綱一樣，說是給多點空白時間讓學生多方探索，刪掉很多重要知識，反而導致後面連結的部分更加吃力。既然16週是趨勢，我們更應該要善用時間為自己做規劃，不要在糜爛中虛度光陰...

**Comment by yyujimmy** [comment]
學術負荷和壓力：16周的學期通常會比較密集，學生可能需要更快地消化課程內容和應付考試。這對於一些學生來說可能會感到壓力更大。但另一方面，較短的學期也意味著學生在一段時間內能夠更集中地學習，而不會分散注意力。
假期時間：有些學生可能更看重寒暑假的時間，希望能有更長的休息和放鬆的機會。因此，減少學期長度以增加假期時間對他們來說可能更有吸引力。
課程規劃和時間管理：16周的學期可能會對學生的時間管理提出更高的要求，因為他們需要更有效地安排學習時間和活動。這對於能夠良好規劃和組織時間的學生來說可能不是問題，但對於其他人來說可能會感到有些壓力。

**Comment by jim1743** [comment]
我比較喜歡16+2週，因為對於某些課程來說，16週就會期末考，比較不會和其他預計在18周期末考的課程撞在一起，讓我有更充餘的時間準備考試的內容，另外若有課程在16週後就放假了，對我來說就更有多餘的時間來做自己想做的事。

**Comment by YOGA012** [comment]
我覺得16週或18週都可以，尤其不要搞什麼16+2，搞得各堂課課程進度安排不一，打亂我學習的步調，但我覺得如果要改制成16週，老師們對於課程的安排也要有所改變，如果還是依照以往18週的授課內容以及步調，那肯定會在接近期末的時候瘋狂趕課跟考試，再加上平時也有作業，這樣會把人給搞瘋。

**Comment by cassy0201** [comment]
個人覺得統一就好，不然考試很容易分配在三周，時間拖很長，到後面反而沒心思念書，且要做其他事也會被考試卡著無法規劃

**Comment by 710388** [comment]
16+2，必修跟通識那些可以分開考比較好

**Comment by yubo0oo** [comment]
我喜歡18周
這樣比較悠閒

**Comment by poiu6fi** [comment]
我覺得18周好一點，不然像這次所有東西都擠在一起有點忙不過來，一兩個禮拜都4.5點才睡了

**Comment by JustAHedgehog** [comment]
救我的看法而言，16週是適用於通識課程的，因為通識課程往不會講得過於深奧，老師也會顧及不同科系的學生而減輕同學課業壓力，因此適用於16週；相反地，18週課程我則認為適用在學生該系的課程，畢竟是身為該課系學生，應該要深入了解該科系內容，且系上選修或必修會常常都是一些課業繁重但極其重要的課程，因此比較適用於18週課程

**Comment by sunset513** [comment]
就我自身的經驗，我認為 18 周是一個比較適合我的制度。最主要的原因是因為 18 周能夠讓期末報告、期末考分散到各周，這能夠大大的減輕我的壓力。並且最後兩周通常會有一點空閒時間，這能夠讓我很好的調整作息，或是做一些自己平常太忙沒空去做的事情。

**Comment by buoioiry** [comment]
我覺得18週對蠻多科系來說可以減少一些考試全部累積在一起的壓力，像我們系上不同科目就有分2-4次考試的，如果壓縮到16週的話會變成期末週非常刺激，很硬的科目也都很有機會在同一週考試，對於修很多科目的同學會壓力很大

**Comment by poyen0806** [comment]
我覺得16+2的制度確實增加了彈性，但同時也帶來更多的不確定性，讓寒假的計畫難以提早安排。相較之下，訂定固定的時間表會帶來更多便利。此外，16+2的安排也可能導致課程和考試集中在同一週的情況發生。總的來說，這就是魚與熊掌不可兼得的抉擇，最終還是以多數決定為準。

**Comment by XiangLuen** [comment]
如果是我，我會偏向 18 週。雖然 16 週會讓寒暑假更長，感覺能多做點事，但實際上學習壓力會更大，尤其是工程類型需要長時間思考的課程，並且有些課程會想要一學期上完全部章節，只有16週會導致課程濃縮太多無法學完。而16+2太雜，像是每堂課的考試時間都不一樣，反而會讓學生有更長時間的壓力

**Comment by SendaiHatsuki** [comment]
我個人是偏向18周，我們系一個學期要學的真得又多又難，期中考周後幾乎每個禮拜都有考試，然後期末還有final project，如果16周，最後一周會非常忙幾乎天天都考試，18周的話至少還多兩個禮拜可以讀書。然後16+2純純低能，根本沒有輕鬆到，反而還卡你時間。

**Comment by LichiNcu** [comment]
我覺得這個問題應該是反思16周的制度從何而來:原本16周是搭配「一學年三學期制」，也就是目前國外大學的主流，16*3=48周 ，但沒記錯的話詳細情況是一學期14周，且國外的大一並沒有像國內「大一國文」、「大一英文」這種類似於高中的基礎學科，選課上更為彈性；兩年或三年就畢業的學生比例也相對較高，但反過來觀察國內的情況: 一學年兩學期制，最多就36周，剩餘的時間便是被寒暑假填滿，美其名是讓學生能夠自行運用，但這變相的是在考驗學生的資訊廣度(能否實習、暑修、抑或是打工，這些選擇並不是單純有想法就能夠獲取資格，有時更像是資訊落差)也許16周或18周並不是問題的核心，而是該去探討如何真正落實國外的節奏跟配置。但是若要改成一學年三學期制，勢必會面臨壓力更大的情況，而且18周能夠傳授的知識量相對偏少，若再刻意片面的效仿國外的「偽16周」，那恐怕只會淪為東施效顰。
綜合目前的制度考量跟學校課程的安排量，我個人認為還是先維持18周比較好，濃縮成16周，只會有更多人得拿假期來養傷(不論是心裡還是身理上的症狀)，不見得就能讓學生變得跟國外人才一樣果決、有行動力、有效率
